docs/deployment-examples.md

```diff
@@ -0,0 +1,331 @@
+# Example Deployment Methods
+There are a number of methods to deploy the OpenMetrics exporter depending on your environment. In this document we explore some of the common features and deployment methods and by providing walk through examples.
+
+## Contents
+ - [Prerequisites](#prerequisites)
+ - [Container Deployment](#container-deployment)
+    - [Container - default - http passing API token with query](#container---default---http-passing-api-token-with-query)
+    - [Container - Tokens File - http with API tokens file embedded in exporter](#container---tokens-file---http-with-api-tokens-file-embedded-in-exporter)
+    - [Container - TLS - https passing API token with query](#container---tls---https-passing-api-token-with-query)
+ - [Executable Binary Deployment](#executable-binary-deployment)
+    - [Binary Specific Prerequisites](#binary-specific-prerequisites)
+    - [Binary - default - http passing API token with query](#binary---default---http-passing-api-token-with-query)
+    - [Binary - Tokens File - http with API tokens file embedded in exporter](#binary---tokens-file---http-with-api-tokens-file-embedded-in-exporter)
+    - [Binary - TLS - https passing API token with query](#binary---tls---https-passing-api-token-with-query)
+
+# Prerequisites
+All deployments will require an API token to authenticate with the array. Read-only only user access is recommended.
+
+1. Generate an API token from your chosen user account or create a new readonly user.
+    API token can be retrieved from either Purity GUI ot CLI.
+
+    ```console
+    pureuser@array01> pureadmin create --role readonly svc-readonly
+    Name          Type   Role    
+    svc-readonly  local  readonly
+
+    pureuser@array01> pureadmin create --api-token svc-readonly
+    Name          Type   API Token                             Created                  Expires
+    svc-readonly  local  a1b2c3d4-e5f6-1234-5678-a1b2c3d4e5f6  2023-11-02 12:00:00 EST  -      
```

Comment:
ensure this is not a real API token - security do checks on all API tokens in Pure repos. Maybe replace with a placeholder.

Reply:
Resolved in commit 4010cfcac1c820121ac3f1816ef1ed4180d0595b